### ISSUE TITLE
Unit tests are broken

### ISSUE THREAD

**Issue by felipeZ** [MD-Studio/MDStudio#22]
the `lietest.py` script reports the following: 
```
Ran 83 tests in 2.229s

FAILED (failures=2, errors=11)
```

**Comment by felipeZ** [comment]
After installing all the dependecies and fixing some minor bugs:
```
======================================================================
ERROR: lie_system.tests.module.system_test (unittest2.loader._FailedTest)
----------------------------------------------------------------------
Traceback (most recent call last):
ImportError: Failed to import test module: lie_system.tests.module.system_test
Traceback (most recent call last):
  File "/home/felipe/miniconda2/envs/mdstudio/lib/python2.7/site-packages/unittest2/loader.py", line 456, in _find_test_path
    module = self._get_module_from_name(name)
  File "/home/felipe/miniconda2/envs/mdstudio/lib/python2.7/site-packages/unittest2/loader.py", line 395, in _get_module_from_name
    __import__(name)
  File "/home/felipe/WorkbenchPython/MDStudio/components/lie_system/lie_system/tests/module/system_test.py", line 10, in <module>
    from lie_system.wamp_tools import wamp_session_schema
ImportError: cannot import name wamp_session_schema


======================================================================
ERROR: test_graph_edge_attribute_get (lie_graph.tests.module.graph_test.TestGraph)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/felipe/WorkbenchPython/MDStudio/components/lie_graph/lie_graph/tests/module/graph_test.py", line 434, in test_graph_edge_attribute_get
    self.assertEqual(self.graph.get((1, 2), key='label'), 'mo')
  File "/home/felipe/WorkbenchPython/MDStudio/components/lie_graph/lie_graph/graph.py", line 684, in get
    if key in target:
TypeError: argument of type 'NoneType' is not iterable

======================================================================
ERROR: test_graph_edge_attribute_set (lie_graph.tests.module.graph_test.TestGraph)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/felipe/WorkbenchPython/MDStudio/components/lie_graph/lie_graph/tests/module/graph_test.py", line 468, in test_graph_edge_attribute_set
    self.graph.attr((1, 2))['tgf'] = 'changed'
TypeError: 'NoneType' object does not support item assignment

======================================================================
ERROR: tearDownClass (lie_logger.tests.module.logger_test.LoggerExportToMongodbObserverTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/felipe/WorkbenchPython/MDStudio/components/lie_logger/lie_logger/tests/module/logger_test.py", line 78, in tearDownClass
    cls.db.stop(terminate_mongod_on_exit=True)
  File "/home/felipe/WorkbenchPython/MDStudio/components/lie_db/lie_db/db_methods.py", line 338, in stop
    pid = self.pid
  File "/home/felipe/WorkbenchPython/MDStudio/components/lie_db/lie_db/db_methods.py", line 205, in pid
    pid = subprocess.check_output(['pgrep', os.path.basename(self.mongod_path)])
  File "/home/felipe/miniconda2/envs/mdstudio/lib/python2.7/posixpath.py", line 114, in basename
    i = p.rfind('/') + 1
AttributeError: 'NoneType' object has no attribute 'rfind'

======================================================================
FAIL: test_not_running_database (lie_db.tests.module.db_test.DatabaseTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/felipe/WorkbenchPython/MDStudio/components/lie_db/lie_db/tests/module/db_test.py", line 59, in test_not_running_database
    self.assertFalse(self.db.isrunning)
AssertionError: True is not false

======================================================================
FAIL: test_start_stop_local_database (lie_db.tests.module.db_test.DatabaseTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/felipe/WorkbenchPython/MDStudio/components/lie_db/lie_db/tests/module/db_test.py", line 43, in test_start_stop_local_database
    self.assertTrue(os.path.exists(self.db._dbpath))
AssertionError: False is not true

----------------------------------------------------------------------
Ran 84 tests in 2.339s

FAILED (failures=2, errors=4, skipped=1)

```

**Comment by PaulVisscher** [comment]
This is fixed in #39 for the core components
